feature-wallet-api/src/main/java/io/novafoundation/nova/feature_wallet_api/presentation/mixin/fee/FeeLoaderMixin.kt

```diff
@@ -76,7 +90,15 @@ interface GenericFeeLoaderMixin<F : GenericFee> : Retriable {
 
         suspend fun setFeeStatus(feeStatus: FeeStatus<F>)
 
-        fun invalidateFee()
+        suspend fun setSupportCustomFee(supportCustomFee: Boolean)
+
+        suspend fun invalidateFee()
+
+        suspend fun commissionChainAsset(): Chain.Asset
+
+        suspend fun commissionAsset(): Asset
```

Comment:
Those could be extensions based on `commissionAssetFlow`. We should keep interfaces as minimal as possible